server/eclipse-project/src/test/java/au/org/democracydevelopers/corla/model/assertion/NEBAssertionTests.java

```diff
@@ -52,7 +53,6 @@
  * -- Scoring of NEB assertions.
  * -- Computation of discrepancies for NEB assertions.
  * -- The logic involved in the re-auditing of ballots.
- * -- TODO: Test Assertion::riskMeasurement()
  * Refer to the Guide to RAIRE for details on how NEB assertions are scored, and how
  * discrepancies are computed (Part 2, Appendix A).
  */
```

Comment:
Again I think you should say (or ref) how you calculated the expected risks. (Because people will assume you just computed them and then wrote these tests to make sure they didn't change, whereas actually you did something much more valuable, which is actually compute what the values are supposed to be rather than making up your tests as a function of what the computer told you.)